project/dexter-vs/Dexter/Core/UI/Settings/SettingsControl.cs

```diff
@@ -198,9 +198,42 @@ DexterInfo IDexterInfoProvider.Load()
             return dexterInfo;
         }
 
+        private void createUser()
+        {
+            // We need to save current url 
+            DexterInfo dexterInfo = GetDexterInfoFromSettings();
+            var result = "";
+
+            if (!validator.ValidateServerConnection(dexterInfo, out result))
+            {
+                MessageBox.Show("Couldn't connect to Dexter server. Setting analysis mode to standalone", "Dexter warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+                standaloneCheckBox.Checked = true;
+                dexterInfo.standalone = true;
+                Save(dexterInfo);
+            }
+            else if (validator.ValidateUserCredentials(dexterInfo, out result))
+            {
+                MessageBox.Show($"Couldn't create user: user {dexterInfo.userName} already exists", "Dexter warning", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+            }
+            else
+            {
+                DexterClient.Instance = new DexterClient(new DexterHttpClientWrapper(dexterInfoProvider), dexterInfoProvider);
```

Comment:
I think it could be refactored - there is no need to have singleton client in case of our plugin at all.